webapp/app/forms/steps/logout_steps.py

```diff
@@ -16,6 +21,20 @@ def __init__(self, **kwargs):
             title=_('form.logout.input-title'),
             intro=_('form.logout.input-intro'),
             form=self.Form,
-            **kwargs,
-            header_title=_('form.logout.header-title'),
-            template='basis/form_full_width.html')
+            **kwargs)
+
+    def render(self, data, render_info):
+        props_dict = LogoutProps(
+            form={
+                'action': render_info.submit_url,
+                'csrf_token': generate_csrf(),
+                'show_overview_button': bool(render_info.overview_url),
+            }
```

Comment:
Can we just omit the hidden field that is in the form?

Reply:
Which hidden field?

Reply:
The Forms attribute has a hidden field that is now deleted